specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2319,6 +2325,25 @@
+      },
```

Comment:
I think this bracket is erroneous

Reply:
good catch, the enum needed to be inside the bracket